oaknational/Oak-Web-Application
# Feat/pupil 680/style programme page b

## Description

Music year: 2011

- List of changes

add style to the programme options page 

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2425--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/physics-secondary-year-11/options
2. 1. Go to https://deploy-preview-2425--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/maths-secondary-year-11/options-l
3. Click on on all buttons and link
4. You should navigate to and from subject and units page

Note: there is a flicker when you go back to programmes page; theres a ticket in the backlog to fix this

### Job Story

**When** I view the ProgrammeListingPage,
**I want to see all the oak styles and components**
**So that the UX is high quality**.

### Implementation details

Create a View folder and implement the the page; using oak components and layouts; 

https://www.figma.com/file/zqpeecgzMGKRli2gUdJOLp/Pupil-Browse?type=design&node-id=4445%3A101982&mode=dev

https://www.figma.com/file/zqpeecgzMGKRli2gUdJOLp/Pupil-Browse?type=design&node-id=4515%3A114797&mode=dev

### Acceptance criteria

- [x]  page matches design
- [x] mobile, tablet, desktop
- [x] screen readable
- [x] tier and examboard logic works correctly
